libs/ledgerjs/packages/cryptoassets/src/tokens.ts

```diff
@@ -23,15 +23,15 @@ const tokensById: Record<string, TokenCurrency> = {};
 const tokensByTicker: Record<string, TokenCurrency> = {};
 const tokensByAddress: Record<string, TokenCurrency> = {};
 const tokensByCurrencyAddress: Record<string, TokenCurrency> = {};
-addTokens(erc20tokens.map(convertERC20));
-addTokens(polygonTokens.map(convertERC20));
-addTokens(trc10tokens.map(convertTRONTokens("trc10")));
-addTokens(trc20tokens.map(convertTRONTokens("trc20")));
-addTokens(bep20tokens.map(convertBEP20));
-addTokens(asatokens.map(convertAlgorandASATokens));
-addTokens(esdttokens.map(convertElrondESDTTokens));
-addTokens(cardanoNativeTokens.map(convertCardanoNativeTokens));
-addTokens(stellarTokens.map(convertStellarTokens));
+addTokens(erc20tokens.map(convertERC20 as any));
```

Comment:
I think we should keep this type safe @haammar-ledger maybe on a later sprint but have all this any is meh

Reply:
sorry I added this as quick fix but maybe we could type it though

Reply:
I will fix

Reply:
@henrily-ledger done